playbooks/roles/ambari-server/tasks/main.yml

```diff
@@ -1,7 +1,4 @@
 ---
-- name: Load default variables
-  include_vars:
-    file: 'defaults/main.yml'
```

Comment:
Actually it seems this one is indeed required (I don't understand yet why)
Without it, deploy fails not finding postgres_port that is configured in those defaults

Reply:
I just reverted above change
ps: removing the `include_vars` from the ambari-blueprint role worked OK